macros/edr/data_monitoring/monitors_query/column_monitoring_query.sql

```diff
@@ -100,7 +102,9 @@
         bucket_start,
         bucket_end,
         bucket_duration_hours,
-        {{- dbt_utils.current_timestamp_in_utc() -}} as updated_at
+        {{- dbt_utils.current_timestamp_in_utc() -}} as updated_at,
+        dimension,
+        dimension_value
```

Comment:
I don't like the fact that we add it on the end of the table.
Is this required because the target table is incremental?

Reply:
Unfortunately yes :(